Test/Units/CommandTestCase.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace atoum\AtoumBundle\Test\Units;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class CommandTestCase extends Test
+{
+    /**
+     * Create a CommandTester instance to test commands
+     *
+     * @param ContainerAwareCommand $command
+     *
+     * @return CommandTester
+     */
+    public function createCommandTester(ContainerAwareCommand $command)
```

Comment:
It's more for an ethical reason than a technical reason : a Command is just a class, and could (should ?) be tested as it.
The CommandTestCase component is usefull for ContainerAwareCommand who are a little bit more complex to test by allowing access to Container.

But i could understand if you want this to be as opened as possible.